<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetActividadUsuario</class>
 <widget class="QWidget" name="WidgetActividadUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>594</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Actividad Usuarios&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QCheckBox" name="chUsuario">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>90</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>90</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Filtrar Usuario</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>89</width>
                         <height>13</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>89</width>
                         <height>13</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Seleccione Usuario</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="cbUsuario"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="chMovimiento">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>108</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>108</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Filtrar Movimiento</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>107</width>
                         <height>13</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>107</width>
                         <height>13</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Seleccione Movimiento</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="cbMovimiento"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QCheckBox" name="chFecha">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>83</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>83</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Filtrar Fecha</string>
                     </property>
                     <property name="tristate">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>82</width>
                       <height>13</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>82</width>
                       <height>13</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Seleccione Fecha</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateEdit" name="deFecha">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>91</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>91</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="dateTime">
                      <datetime>
                       <hour>0</hour>
                       <minute>0</minute>
                       <second>0</second>
                       <year>2000</year>
                       <month>1</month>
                       <day>1</day>
                      </datetime>
                     </property>
                     <property name="maximumTime">
                      <time>
                       <hour>23</hour>
                       <minute>59</minute>
                       <second>59</second>
                      </time>
                     </property>
                     <property name="calendarPopup">
                      <bool>true</bool>
                     </property>
                     <property name="timeSpec">
                      <enum>Qt::LocalTime</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="chArticulo">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>90</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>90</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Filtrar Artículo</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>89</width>
                         <height>13</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>89</width>
                         <height>13</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Seleccione Artículo</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="cbArticulo"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="chSinFiltros">
                   <property name="text">
                    <string>Sin filtros</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pbAplicar">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Buscar</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbBorrar">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>121</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>121</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Eliminar</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QTableView" name="tvActividad">
             <property name="minimumSize">
              <size>
               <width>790</width>
               <height>345</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>790</width>
               <height>345</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideMiddle</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chUsuario</tabstop>
  <tabstop>cbUsuario</tabstop>
  <tabstop>chMovimiento</tabstop>
  <tabstop>cbMovimiento</tabstop>
  <tabstop>chFecha</tabstop>
  <tabstop>deFecha</tabstop>
  <tabstop>chArticulo</tabstop>
  <tabstop>cbArticulo</tabstop>
  <tabstop>pbAplicar</tabstop>
  <tabstop>pbBorrar</tabstop>
  <tabstop>tvActividad</tabstop>
  <tabstop>chSinFiltros</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
